lib/wings.rb

```diff
@@ -118,6 +118,12 @@ def self.inspect
   Hyrax::AccessControl.new(**attrs)
 end
 
+ActiveTriples::Properties::ClassMethods.send(:define_method, :protected_property_name?) do |name|
```

Comment:
merged something similar upstream and released as ActiveTriples 1.1.1.

new builds should pick this up automatically